docs/learn/architecture.md

```
@@ -9,16 +9,13 @@ At the core of **Polymer’s security model** is the aggregation of **rollup sta
 
 > Every rollup connected to Polymer maintains an always-updated awareness of the state of all other rollups within the network.  
 
-#### **Why State (Block Headers)?**
-
+### **Why State (Block Headers)?**
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)